docs/_sidebar.md

```diff
@@ -12,7 +12,8 @@
 [Dependencies](structure/dependencies.md) <br>
 [Variables](structure/variables.md) <br>
 &nbsp;&nbsp;&nbsp;&nbsp;[Modifiers](structure/variables/modifiers.md) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;[Control Flow](structure/variables/control.md)
+&nbsp;&nbsp;&nbsp;&nbsp;[Control Flow](structure/variables/control.md) <br>
+[Schemas](structure/schemas.md)
```

Comment:
would put this under the "Working with Pusta" category (and then also move the md file to the corresponding folder)